types/forge-viewer/index.d.ts

```diff
@@ -21,7 +21,9 @@
 //
 /// <reference types="THREE" />
 
-declare namespace Autodesk {
+import * as THREE from 'three';
+
+export namespace Autodesk {
```

Comment:
I'm not sure about this. It's possible but if I try it in existing project I'm actually getting issues later when building the application (i.e. webpack is looking for forge-viewer npm module which obviously doesn't exist).

Reply:
Found a way to revert the `declare namespace Autodesk`, please have a try!